packages/realm-web/src/User.ts

```diff
@@ -346,6 +347,10 @@ export class User<
         throw new Error("Not yet implemented");
     }
 
+    public remoteMongoClient(serviceName: string): Realm.Services.MongoDB {
```

Comment:
Wasn't `remoteMongoClient` renamed to `mongoClient` at some point?

Reply:
Yep - good catch. I saw it and forgot all about it :D